Sources/PointFree/Database.swift

```diff
@@ -150,36 +149,6 @@ private func migrate() -> EitherIO<Error, Prelude.Unit> {
     .map(const(unit))
 }
 
-#if os(iOS)
```

Comment:
we dont run tests on ios anymore (now that snapshots work on macOS) so removing this